src/plugins/recordTypes/location/vocabularies.js

```diff
@@ -0,0 +1,22 @@
+import { defineMessages } from 'react-intl';
+
+export default {
+  location: {
+    messages: defineMessages({
+      name: {
+        id: 'vocab.location.location.name',
+        description: 'The name of the vocabulary.',
+        defaultMessage: 'Storage locations',
+      },
+      collectionName: {
+        id: 'vocab.location.location.collectionName',
+        description: 'The name of a collection of records from the vocabulary.',
+        defaultMessage: 'Storage locations',
+      },
+    }),
+    serviceConfig: {
+      servicePath: 'urn:cspace:name(location)',
+    },
+    sortOrder: 0,
+  },
+};
```

Comment:
Don't need to add a vocabulary. Just change the name of the local vocabulary in messages.js.